cmd/osm-bootstrap/osm-bootstrap.go

```diff
@@ -434,13 +434,6 @@ func (b *bootstrap) createMeshRootCertificate() error {
 
 	createdMRC.Status = configv1alpha2.MeshRootCertificateStatus{
```

Comment:
Can we add TODOs about creating the MRC in osm-boostrap per the [gist](https://gist.github.com/keithmattix/9ac73abbcb5721b0ce58102ac3ebff29)

Reply:
done